R/agglomerate.R

```diff
@@ -76,10 +76,12 @@
 #' @param altexp String or integer scalar specifying an alternative experiment
 #'   containing the input data.
 #'
-#' @param strip_altexp \code{TRUE} or \code{FALSE}: Should alternative
+#' @param strip.altexp \code{TRUE} or \code{FALSE}: Should alternative
```

Comment:
Why not `rm.altexp`?

Reply:
Same thing: should it be the  `altexp.rm`

Reply:
ok if we always have .rm in the end - I am not sure which way is optimal. The rm. in front would also have certain clarity. You can decide.

Reply:
It think we can use *rm everywhere.

--> `altexp.rm`

@Daenarys8 can you check that these parameters that remove something follows this same scheme?

Reply:
ACK.